k8s/cassandra/apptest/deployer/manifest/tester.yaml.template

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
```

Comment:
We should use a `Job` here.

Maybe we should also set a deadline on it? https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#job-termination-and-cleanup. I understand that the test itself already has a deadline. But we might want to set a deadline on the job is to make it more self-contained. Can use a large timeout here as a fail-safe.

Reply:
Added timeout (20 minutes) as a fail-safe.